### ISSUE TITLE
Bug: Right click -> "Select all" results in the query window is very slow

### ISSUE THREAD

**Issue by Pharaoh2k** [avan06/PS4CheaterNeo#33]
![Hm53e2v](https://github.com/avan06/PS4CheaterNeo/assets/5753365/07ae3885-a306-4e9d-8d65-4d4f54f10303)

If I select the first and SHIFT+click the last, all results are getting selected instantly.
But right click -> select all takes a long time on these (1,000,000) results.

**Comment by Pharaoh2k** [comment]
The quantity is not the point and it's obviously a performance issue if SHIFT+click works instantly, while Select All doesn't.

**Comment by avan06** [comment]
Regarding the efficiency issue with "Select all," it falls somewhere between what can be changed and what cannot.

Firstly, the "SHIFT+click" selection method isn't a feature I implemented; it's actually built into the ListView itself.

Secondly, the "Select all" option in the menu is something I programmed. It involves a for loop to set the item statuses to selected.

The speed difference between these two methods is quite significant, and I find it interesting. After researching related web resources, I've found that there isn't a way to improve the efficiency of the for loop "Select all" in C#. However, there is a solution: using the traditional Win32 SendMessage function can greatly improve it.
https://stackoverflow.com/a/1118396
